sizefashion/src/context/products_context.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext, useEffect, useReducer } from 'react'
+import reducer from '../reducers/products_reducer'
+import {
+  GET_PRODUCTS_BEGIN,
+  GET_PRODUCTS_SUCCESS,
+  GET_PRODUCTS_ERROR,
+  GET_SINGLE_PRODUCT_BEGIN,
+  GET_SINGLE_PRODUCT_SUCCESS,
+  GET_SINGLE_PRODUCT_ERROR,
+} from '../actions'
+import { productDataType } from '../utils/productData'
```

Comment:
@otaaviostein make sure your types are all capitalized (`ProductDataType` instead of `productDataType`)